assets/ts/schedule/schedule-loader.tsx

```diff
@@ -59,101 +29,156 @@ const updateURL = (origin: SelectedOrigin, direction?: DirectionId): void => {
   }
 };
 
-export const renderAdditionalLineInformation = (
-  schedulePageData: SchedulePageData
-): void => {
-  const { schedule_note: scheduleNote } = schedulePageData;
+const getPageData = (): {
+  schedulePageData: SchedulePageData;
+  branchesAreEmpty: boolean;
+  mapData: MapData;
+} => {
+  const schedulePageDataEl = document.getElementById("js-schedule-page-data");
+  const mapDataEl = document.getElementById("js-map-data");
+  if (!schedulePageDataEl || !mapDataEl) {
+    throw new Error(
+      `Page Elements are Missing schedulePageDataEl:${schedulePageDataEl} mapDataEl:${mapDataEl}`
+    );
+  }
+  const schedulePageData = JSON.parse(
+    schedulePageDataEl.innerHTML
+  ) as SchedulePageData;
+  const branchesAreEmpty =
+    schedulePageDataEl.getAttribute("data-branches-are-empty") === "true";
+  const mapData: MapData = JSON.parse(mapDataEl.innerHTML);
+
+  return {
+    schedulePageData,
+    branchesAreEmpty,
+    mapData
+  };
+};
 
+const render = (): void => {
+  const { schedulePageData, branchesAreEmpty, mapData } = getPageData();
+
+  const { direction_id: directionId } = schedulePageData;
+
+  createScheduleStore(directionId);
   ReactDOM.render(
-    <Provider store={store}>
-      <ScheduleLoader
-        component="ADDITIONAL_LINE_INFORMATION"
-        schedulePageData={schedulePageData}
-        updateURL={updateURL}
-      />
-    </Provider>,
-    document.getElementById("react-root")
+    doRender(schedulePageData, branchesAreEmpty, mapData),
+    document.getElementById("react-root-schedule-page")
   );
-  // don't show Schedule Finder for subway
-  if (scheduleNote) {
-    ReactDOM.render(
-      <Provider store={store}>
-        <ScheduleLoader
-          component="SCHEDULE_NOTE"
-          schedulePageData={schedulePageData}
-          updateURL={updateURL}
-        />
-      </Provider>,
-      document.getElementById("react-schedule-note-root")
-    );
-  } else {
-    const scheduleFinderRoot = document.getElementById(
-      "react-schedule-finder-root"
-    );
-    if (scheduleFinderRoot) {
-      ReactDOM.render(
-        <Provider store={store}>
-          <ScheduleLoader
-            component="SCHEDULE_FINDER"
-            schedulePageData={schedulePageData}
-            updateURL={updateURL}
-          />
-        </Provider>,
-        scheduleFinderRoot
-      );
-    }
-  }
 };
 
-const renderDirectionAndMap = (
+export const getDirectionAndMap = (
```

Comment:
as far as I can tell this function isn't called outside this file, so no need to export